PhoneBook.Doc415/DataAccess.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PhoneBook.Doc415.Context;
+using PhoneBook.Doc415.Models;
+
+namespace PhoneBook.Doc415;
+
+internal class DataAccess
+{
+    public void AddContact(string _name, string _email, string _phone, string _title)
+    {
+        Contact newContact= new Contact();
+        newContact.Name= _name;
+        newContact.Email= _email;
+        newContact.Title= _title;
+        newContact.PhoneNumber= _phone;
+
+        try
+        {
+            var db = new PhoneBookContext();
```

Comment:
Why are you creating context in method? Why not to use Inversion of Control? (assign this context in constructor and use private variable)

Reply:
What yoı suggest is connected scenario
I used diaconnected scenario. Uses less resourcess and does not leave open connection to db. Better for remote servers. Any of them can be used. I prefered this one.

Reply:
my suggestion it's not about performance but about maintainability and testability.
Another thing is, that this is not "connected" scenario. 
As you are using EF it's open/close connection on request itself. Not when context it's created.

Reply:
Hm ok i am new to this. I ll review it again. Thank you.

Reply:
ok i understand what you mean .